README.md

```diff
@@ -7,43 +7,45 @@ Rill Developer makes it effortless to transform your datasets with SQL and creat
 - *no "run query" button required* – responds to each keystroke by re-profiling the resulting dataset
 - *radically simple dashboards* - thoughtful, opinionated defaults to help you quickly derive insights from your data
 
-## Pick an install option:
-Want to try for yourself? You can get started in less than 2 minutes with our installation script (Mac and Linux):
-```
-curl -s https://cdn.rilldata.com/install.sh | bash
-```
+![Kapture 2022-07-21 at 15 34 45](https://user-images.githubusercontent.com/5587788/180313797-ef50ec6e-fc2d-4072-bb77-b2acf59205d7.gif "732257485")
 
-![Kapture 2022-07-21 at 15 34 45](https://user-images.githubusercontent.com/5587788/180313544-a7a03fcc-ea85-4b51-8ec3-ad5ed5043571.gif)
+## Installation
 
+On macOS, we recommend installing `rill` using Brew:
 
-## We want to hear from you
-
-You can [file an issue](https://github.com/rilldata/rill-developer/issues/new/choose) directly in this repository or reach us in our [Rill discord](https://bit.ly/3unvA05) channel. Please abide by the [rill community policy](https://github.com/rilldata/rill-developer/blob/main/COMMUNITY-POLICY.md).
-
-Our binary script is the fastest way to install Rill Developer on Mac or Linux. You can install it using this command:
+```bash
+brew install rilldata/rill-developer/rill
+```
 
-- [script](https://docs.rilldata.com/install/binary) : run the following script and you are ready to start using the Rill [CLI](https://docs.rilldata.com/cli).
+On Linux, we recommend installing `rill` using our installation script:
 
-```
+```bash
 curl -s https://cdn.rilldata.com/install.sh | bash
 ```
 
-## Quick start a new project
+<!-- TODO: Add docs link here -->
 
-You can create and augment your own projects in Rill Developer using the [CLI](https://docs.rilldata.com/cli). Or simply start the application and enrich your project from there:
+## Quick start
 
-```
+You start Rill using the CLI. To start Rill in a new, empty project:
+
+```bash
+rill init --project my-project
+cd my-project
 rill start
 ```
 
-You can also try our example:
+You can also check out the Rill Developer repository and explore one of our example projects. For example:
 
+```bash
+git clone https://github.com/rilldata/rill-developer.git
+cd rill-developer/examples/sf_props
+rill start
```

Comment:
From implementation perspective, github workflow around sharing and rehydrating artifacts via a github repo should be working already. 
@magorlick : This is a small example starting from a project directory having artifacts that is hosted on github. 
Is this is what you were expecting for an example project ? 

Feel free to take charge and modify the docs as well to add more elaborative steps.

Reply:
There's a repo here now we can use instead: https://github.com/rilldata/rill-developer-example

Reply:
Updated to reflect this